go-controller/pkg/types/const.go

```diff
@@ -166,4 +169,5 @@ const (
 
 	// different secondary network topology type defined in CNI netconf
 	Layer3Topology = "layer3"
+	Layer2Topology = "layer2"
```

Comment:
I worry a bit about the value of the configuration - especially since a typo can get a valid - but wrong / undesired configuration. 

I think this would be less error-prone if we had used names like "routed/switched/localnet". Can we still do anything about this ?

Reply:
discussed in the meeting.